test/jasmine/tests/plots_test.js

```diff
@@ -248,7 +248,7 @@ describe('Test Plots', function() {
     describe('Plots.resize', function() {
         var gd;
 
-        beforeEach(function(done) {
+        beforeAll(function(done) {
```

Comment:
Hmm… `beforeAll`? Usually when I've written that, it's not actually what I want. Does it leak state between the tests? Is it *definitely* what you meant here?